roles/confluent.common/tasks/redhat.yml

```diff
@@ -22,6 +23,7 @@
     gpgcheck: "{{confluent_common_repository_redhat_main_gpgcheck}}"
     gpgkey: "{{confluent_common_repository_redhat_main_gpgkey}}"
     enabled: "{{confluent_common_repository_redhat_main_enabled}}"
+    proxy: "{{proxy_env.http_proxy}}"
```

Comment:
Do you want to use `environment` to make it consistent and usable with different protocols

Reply:
i think this module write to a .repo file 

so the proxy argument will end up as a line in that file

https://docs.ansible.com/ansible/latest/collections/ansible/builtin/yum_repository_module.html

Reply:
Got it!